### PR TITLE
GrafanaUI: Add two icons add-user and clip-attach

### PR DESCRIPTION
**Pull request by reemtariqq**
**What is this feature?**

Add two icons to the unicons` add-user `and` clip-attach`
for more : [Slack thread](https://raintank-corp.slack.com/archives/CT7S455SP/p1713962582627559)

Screenshot 2024-04-30 at 10 53 11

Screenshot 2024-04-30 at 10 53 05

**Special notes for your reviewer:**

Please check that:
- [x] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: packages/grafana-data/src/types/icon.ts [hunk 1]
```diff
@@ -245,6 +245,8 @@ export const availableIconsIndex = {
   'wrap-text': true,
   rss: true,
   x: true,
+  'add-user': true,
+  'clip-attach': true,
```
Comment (joshhunt) [inline packages/grafana-data/src/types/icon.ts:249]: i think `paperclip` is a better name, but 🤷

**Review by joshhunt — APPROVED** [review]

File: packages/grafana-data/src/types/icon.ts (hunk 1 above)
Comment (Aires-Grafana) [reply packages/grafana-data/src/types/icon.ts:249]: I see you @joshhunt. Same thought about the "add-user" but it is named from the Unicon official library. 🤷🏻‍♂️

Comment (anavss) [reply packages/grafana-data/src/types/icon.ts:249]: hey folks 👋🏼 actually I just checked and the names of these icons in Unicon is `add-account` and `attach`. I think the names we are putting forward are the best ones for discoverability, but feel free if you'd like to rename - no big deal.

Comment (joshhunt) [reply packages/grafana-data/src/types/icon.ts:249]: I don't feel strongly about this (hence why I approved - you can merge whenever you're ready), but either paperclip or attach seem like the best options to me. add-user is fine to me.

**Pull request merged**